samcli/local/init/templates/cookiecutter-aws-sam-hello-ruby/{{cookiecutter.project_name}}/README.md

```diff
@@ -124,24 +114,22 @@ aws cloudformation describe-stacks \
 
 ## Testing
 
-We use [Mocha](http://gofreerange.com/mocha/docs) for testing our code and you can install it using gem: ``gem install mocha`` 
-
-Next, we run our initial unit tests:
+Run our initial unit tests:
 
 ```bash
-ruby tests/unit/test_hello.rb
+ruby tests/unit/test_handler.rb
 ```
 
 **NOTE**: It is recommended to use a Ruby Version Manager to manage, and work with multiple ruby environments from interpreters to sets of gems
 # Appendix
 
 ## AWS CLI commands
 
-AWS CLI commands to package, deploy and describe outputs defined within the cloudformation stack:
+AWS CLI commands to package, deploy and describe outputs defined within the cloudformation stack after building:
 
 ```bash
 sam package \
-    --template-file template.yaml \
+    --template-file .aws-sam/build/template.yaml \
```

Comment:
Need to investigate this, thought there was some code that automatically picked this up.

Reply:
No need to add `--template-file` when using build anymore. We will auto pick up build if it exists otherwise default to `template.yaml`.

https://github.com/awslabs/aws-sam-cli/blob/develop/samcli/commands/package/__init__.py#L21
and 
https://github.com/awslabs/aws-sam-cli/blob/develop/samcli/commands/_utils/options.py#L35